debian/pt-web-portal.install

```diff
@@ -1,7 +1,5 @@
-pkg/pwdptwp.sh                              /usr/lib/pt-web-portal
-src/expand-fs.sh                            /usr/lib/pt-web-portal
-src/notify-on-port-busy.sh                  /usr/lib/pt-web-portal
-src/server/Pipfile*                         /usr/lib/pt-web-portal
-src/server/run.py                           /usr/lib/pt-web-portal
-src/server/backend                          /usr/lib/pt-web-portal
+debian/extras/notify-on-port-busy.sh        /usr/lib/pt-web-portal
+debian/extras/pwdptwp.sh                    /usr/lib/pt-web-portal
 src/miniscreen                              /usr/lib/pt-web-portal
+src/server/backend                          /usr/lib/pt-web-portal
```

Comment:
This may not need to be defined explicitly - can we get this to be installed as a `pt_web_portal` package?